lightkurve/targetpixelfile.py

```diff
@@ -569,10 +568,20 @@ def interact(self, lc=None, notebook_url='localhost:8888', max_cadences=30000):
         max_cadences : int
             Raise a RuntimeError if the number of cadences shown is larger than
             this value. This limit helps keep browsers from becoming unresponsive.
+        aperture_mask: np.ndarray
+            An aperture mask to be used as the starting mask at the beginning of
+            a new .interact() visualization. Defaults to the Kepler Pipeline Mask.
```

Comment:
I recommend copying the docstring for `aperture_mask` from `tpf.extract_aperture_photometry()` or `tpf._parse_aperture_mask()` and pasting it here.  I think it will benefit clarity if every function that accepts an `aperture_mask` as a parameter accepts exactly the same options and has exactly the same docstring.